nuxt.config.ts

```diff
@@ -111,6 +110,9 @@ export default defineNuxtConfig({
 
     postcss: {
         plugins: {
+            '@csstools/postcss-global-data': {
+              files: ["./app/assets/css/media.css"]
```

Comment:
Does this work?
              files: ["~/assets/css/media.css"]

Reply:
No `~` and `@` do not work 

```
Pre-transform error: [postcss] ENOENT: no such file or directory, open '/Users/lukewaldner/Documents/GitHub/fuxt/~/app/assets/css/media.css' 

Pre-transform error: [postcss] ENOENT: no such file or directory, open '/Users/lukewaldner/Documents/GitHub/fuxt/@/app/assets/css/media.css'
```